### PR TITLE
added sending and estimating states to useEtherspotTransactions hook

### PR DESCRIPTION
**Pull request by poocart**
## Description

- Added `isSending`, `isEstimating`, `containsSendingError`, `containsEstimatingError` to `useEtherspotTransactions` hook

## How Has This Been Tested?

- Locally ran example dapp, integration tests to be updated.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@etherspot/transaction-kit",
   "description": "React Etherspot Transaction Kit",
-  "version": "0.6.12",
+  "version": "0.6.13",
```
Comment (IAmKio) [inline package.json:4]: Minor point but this feels like a minor upgrade as we're introducing new hook variables?

**Review by IAmKio — APPROVED** [review]
Just one minor comment but it's not a showstopper

File: package.json (hunk 1 above)
Comment (poocart) [reply package.json]: Updated to `0.7.0`

File: package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@etherspot/transaction-kit",
   "description": "React Etherspot Transaction Kit",
-  "version": "0.6.12",
+  "version": "0.7.0",
```
Comment (IAmKio) [inline package.json:4]: Ty 🙏🏾

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**